scripts/benchmarks/merge-benchmark-data.js

```diff
@@ -0,0 +1,43 @@
+const BENCHMARK_LAYERS = [
+  '../../staging/benchmarks/benchmarks_base_2017.json',
+  '../../staging/benchmarks/benchmarks_updates_20171109.json',
+  '../../staging/benchmarks/benchmarks_readmission_20171109.json'
```

Comment:
Why is the readmission one separate? Could you document the process for adding new benchmarks in the future? It looks like you ran parse-benchmarks-data to generate these files?

Reply:
I wonder if we could make this step a little less hardcoded - so iterate though the files in the benchmarks directory and merge based no date stamps?

Reply:
Yeah, I'll add more info in the CONTRIBUTING file on how to add new benchmarks.

I'm thinking every time we need to add or modify a benchmark, we should add a new json file for auditing and regeneration incase there's a bug. Ya, I used parse-benchmarks-data and I'll document how to use that too.

We should make the ordering explicit because ordering matters when we merge. We could make it less hard coded by appending 001-<name>, 002-<name> and have ordering by file name or include in the code the ordering as I'm doing it now.

WDYT?

Reply:
I think because we're doing full replacement, there isn't 'modifying' a benchmark, just replacing with more recent copies, which means ordering will matter. Explicit ordering via dates in filenames makes sense to me (which could get parsed in this script)